src/components/Atoms/Pagination/List/List.style.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+import List from './List';
+
+const StyledList = styled(List)`
+  ul {
```

Comment:
As comment above this could be styled components `StyledList`(ul) and `ListItems`(li).

Let's use flex-box instead float.

Reply:
Fixed